isvcs/zk.go

```diff
@@ -232,7 +233,72 @@ func zkFourLetterWord(server, command string, timeout time.Duration) ([]byte, er
 	return resp, nil
 }
 
-func BuildZookeeperInstance(instanceID int) dao.RunningService{
+type ZooKeeperServerStats struct {
+	InstanceID  int
+	Connections int
+	Mode        string
+}
+
+func GetAllZooKeeperServerStats() []ZooKeeperServerStats {
+	stats := []ZooKeeperServerStats{}
+	for _, key := range GetZooKeeperKeys() {
+		stats = append(stats, GetZooKeeperServerStats(key.InstanceID, key.Connection))
+	}
+
+	return stats
+}
+
+func GetZooKeeperServerStatsByID(instanceID int) ZooKeeperServerStats {
+	for _, key := range GetZooKeeperKeys() {
+		if key.InstanceID != instanceID {
+			continue
+		}
+
+		return GetZooKeeperServerStats(instanceID, key.Connection)
+	}
+	return ZooKeeperServerStats{}
+}
+
+func GetZooKeeperServerStats(instanceID int, connection string) ZooKeeperServerStats {
```

Comment:
We need to split out this logic somewhat like we do for health statuses.

In the current impl, a UI request to the CC backend turns around an issues a series of 4-letter commands to each ZK instance.   Instead, we need a go routine running in the background to continually sample the ZK stats, say every 30 seconds, and storing that info in some kind of internal map/list which the UI reads from.